docs/drivers.md

```diff
@@ -3,26 +3,98 @@ Bit components are language agnostic. You can write components in any language y
 
 ### What does the driver do?
 
-The driver resolves the ID of the components you want to use in your code, and using [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection) run the component you want to use.
+Drivers are responsible for several tasks:
+
+1. Resolving the `bit.json` file, and fetching all components and dependencies to the `components` directory.
+2. Creating and maintainig all relevant links for the language-specific development environment, with the components imported with Bit.
+  * For example - in JavaScript, to use a module via `require`, the module will need to be in the `node_modules` direcotry, so in this case the driver maintains a mock-module with links back to the `components` directory, where all components are stored.
 
```

Comment:
> Resolving the `bit.json` file, and fetching all components and dependencies to the `components` directory.

Importing all the component dependencies according to the bit.json file.

>  with the components imported with Bit.

of the components imported/created/managed by Bit.

> via `require`

via `import`/`require`

> a mock-module

dynamic-module